android/app/react.gradle

```diff
@@ -68,7 +68,7 @@ task bundleReleaseJsAndAssets(type: Exec) {
             entryFile, "--bundle-output", jsBundleFileRelease, "--assets-dest", resourcesDirRelease
     } else {
         commandLine "react-native", "bundle", "--platform", "android", "--dev", "false", "--entry-file",
-            entryFile, "--max-workers", "2", "--bundle-output", jsBundleFileRelease, "--assets-dest", resourcesDirRelease
+            entryFile, "--bundle-output", jsBundleFileRelease, "--assets-dest", resourcesDirRelease, (System.getenv("CI") == "true") ? "-j2" : ""
```

Comment:
Nitpick: `System.getenv("CI") ?: "" != ""`?

Reply:
Is that what the condition should be or the entire ternary?

Reply:
The entire ternary. ?: is the "Elvis" operator; if the lhs doesn't exist / is falsy, it uses the right side. 

Bah. That's not really what I want. I want System.hasEnv, but I don't think that exists.